system_baseline/views/v1.py

```diff
@@ -246,6 +246,30 @@ def get_baselines(limit, offset, order_by, order_how, display_name=None):
     )
 
 
+@metrics.baseline_fetch_all_requests.time()
+@metrics.api_exceptions.count_exceptions()
+def get_baselines_by_system_id(system_id=None):
```

Comment:
Can we have the shared regular views for an internal api?  The mgmt_api has a mgmt_views/v0.py separately, so I was expecting this to work similarly; separate schema, separate view.

Reply:
Yes, you are right.

Reply:
I split views and tests to have internal parts of API in it's own files